python/src/spark_rapids_ml/knn.py

```diff
@@ -493,3 +526,72 @@ def _get_cuml_transform_func(
         raise NotImplementedError(
             "'_CumlModel._get_cuml_transform_func' method is not implemented. Use 'kneighbors' instead."
         )
+
+    def exactNearestNeighborsJoin(
+        self,
+        query_df: DataFrame,
+        distCol: str = "distCol",
+    ) -> DataFrame:
+        """
+        This function returns the k exact nearest neighbors (knn) in item_df of each query vector in query_df.
```

Comment:
Should clarify what item_df is now that it has been removed as an argument here.

Reply:
Added a one-sentence description.